.github/docker/vulcanexus/Dockerfile

```
@@ -20,10 +20,9 @@ RUN echo "Docker Base image used: ${docker_image_base}"
 RUN source "/opt/vulcanexus/$VULCANEXUS_DISTRO/setup.bash" && \
         apt-get update && \
         apt-get install -y \
-            ros-humble-example-interfaces \
-            ros-humble-example-interfaces \
-            ros-humble-demo-nodes-cpp \
-            ros-humble-demo-nodes-py
+            ros-$VULCANEXUS_DISTRO-example-interfaces \
```

Vulcanexus and ROS 2 distros should be always the same, so we can unify both environment variables:


```suggestion
            ros-$ROS_DISTRO-example-interfaces \
```